apps/apple/FluentUITester/ios/FluentUITester/AppDelegate.m

```diff
@@ -48,7 +32,11 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 
 - (NSURL *)sourceURLForBridge:(RCTBridge *)bridge
 {
+#if DEBUG
   return [[RCTBundleURLProvider sharedSettings] jsBundleURLForBundleRoot:@"index" fallbackResource:nil];
+#else
+  return [[NSBundle mainBundle] URLForResource:@"main" withExtension:@"jsbundle"];
+#endif
```

Comment:
Is this common to be different between release and debug? If so, why wouldn't that be a react-native implementation detail?